docs/run-platform-server/prerequisites.mdx

```diff
@@ -3,56 +3,45 @@ title: Prerequisites
+**_PostgreSQL Database Server Compute Instance:_**
 
-However, the requirements will vary depending on your chosen retention period and desired catch-up speed. Note that **most operators will not need full history**, and as the network continues to grow, tracking full history will become increasingly prohibitive. As of late 2021, DB storage to support historical retention is growing at a rate of 0.8 TB / month. It is highly recommended to configure retention of only the history needed to support your functionality.
+- CPU: 4 cores
+- RAM: 32 GB
+- Storage: SSD with a capacity of 2 TB (NVMe or Direct Attached Storage) capable of handling at least 7K IOPS (I/O operations per second).
 
-#### Requirements
+Please note that a minimum of PostgreSQL version 12 is required.
 
-Minimally, your disk storage type **must** be an SSD (e.g. NVMe, Direct Attached Storage) and your I/O **must** handle >15k iops (I/O operations per second). The following table breaks down hardware specifications for ingestion at different retention levels and performance tiers.
+These specifications assume a 30-day retention window for data storage. For a longer retention window, the system requirements will be higher. For more information about data ingestion, history retention, and managing storage, check the [ingestion](./ingestion.mdx) section.
 
-Note that each component can be scaled independently and for redundancy, in the manner of traditional _n_-tier systems which is covered later in [Scaling](./scaling.mdx). Ingestion can be sped up via configuring more Captive Core parallel workers (requiring more compute and RAM).
+**Multiple Instance Deployment**
 
-| Component |  | Retention Period |
-| :-- | :-- | :-- | --- |
-|  | **30 days** | **90 days** | **Full History** |
-| **Parallel worker count**<br/>(est. ingestion time) | 6 workers (1 day) | 10 workers (1 day) | 20+ workers (2 days) |
-| **Horizon** | **CPU**: 10 cores (min: 6) <br/>**RAM**: 64 GB (min: 32)<br/> | **CPU**: 16 (min: 8)<br/> **RAM**: 128 GB (64)<br/> | **CPU**: 16 (10)<br/> **RAM**: 512 GB (256) |
-| **Database** | **CPU**: 16 cores (min: 8)<br/>**RAM**: 64 GB (min: 32GB)<br/>**Storage**: 2 TB<br/>**IOPS**: 20K (min: 15K) | **CPU**: 16 (12)<br/>**RAM**: 128 GB (64)<br/>**Storage**: 4 TB<br/>**IOPS**: 20K (15K) | **CPU**: 64 (32)<br/>**RAM**: 512 GB (256)<br/>**Storage**: 10 TB<br/>**IOPS**: 20k (15k) |
-| **Storage**<br/>(all same) |  | **SSD** (NVMe, Direct Attached Storage preferred) |  |
-| **AWS**<br/>(reference) | **Captive Core**: `m5.2xlarge`<br/>**Database**: `r5.2xlarge` | **Captive Core**: `m5.4xlarge`<br/>**DB**: `r5.4xlarge` | **Captive Core**: `c5.2xlarge` (x2)<br/>**DB**: `r5.16xlarge` (ro)<br/>`r5.8xlarge` (rw) |
+To achieve high availability, redundancy, and high throughput, explore the [scaling](./scaling.mdx) strategy. It provides detailed prerequisites and guidelines to determine the appropriate [number of Horizon instances](./configuring.mdx#multiple-instance-deployment) to deploy.
```

Comment:
hmm, do we need to update scaling.mdx, it hasn't been touched as of the rest of partner experience tickets, maybe it falls under the purview of prerequisites?

Reply:
Ok, I'll take a look at the scaling doc.